### PR TITLE
feat(mgpath): add FromMageRoot

### PR DESCRIPTION
**Pull request by odsod**

### CODE REVIEW

**Review by Edholm — APPROVED** [review]
Neat :camera_flash:

File: cmd/build/main.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ var mgmakeGen []byte
 func main() {
 	ctx := logr.NewContext(context.Background(), mglogr.New("mage-tools-build"))
 	logr.FromContextOrDiscard(ctx).Info("building binary and generating Makefiles...")
-	mageDir := mgpath.FromGitRoot(mgpath.MageDir)
+	mageDir := mgpath.FromMageRoot()
```
Comment (alethenorio) [inline cmd/build/main.go:25]: What's the mage root? The same root as the git root?

**Comment by alethenorio** [comment]
Not sure I understand the concept of mage "root". Is this the .mage folder inside a repository? If so should we maybe change the word "root" as it doesn't quite seem to fit?

File: cmd/build/main.go (hunk 1 above)
Comment (odsod) [reply cmd/build/main.go:25]: It's the root dir where all the mage related files are kept - currently hard-coded to `.mage`

**Comment by odsod** [comment]
@alethenorio See inline comment - open to alternate naming suggestions!

**Comment by alethenorio** [comment]
Haven't gotten alternatives. Just feels that the name "Root" doesn't quite fit for a dot folder inside the repo.

Maybe "FromDotMageDir" but that's a little too long.

**Review by alethenorio — APPROVED** [review]

**Comment by viktorvoltaire** [comment]
Isn’t `FromMageDir` reasonable then?

**Review by viktorvoltaire — APPROVED** [review]
LGTM

**Pull request merged**